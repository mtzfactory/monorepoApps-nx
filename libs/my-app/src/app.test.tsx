import * as React from 'react';
import { render, screen } from '@testing-library/react-native';

import { App } from './app';

test('renders correctly', () => {
  const expectedText = 'Lorem ipsum';

  render(<App />);

  const text = screen.getByText(expectedText);

  expect(text).toHaveTextContent(expectedText);

  expect(screen.toJSON()).toMatchSnapshot();
});
