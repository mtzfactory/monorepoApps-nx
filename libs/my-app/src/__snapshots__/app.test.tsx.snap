// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Object {
      "backgroundColor": "#00f0f0",
      "flex": 1,
    }
  }
>
  <View
    style={
      Array [
        Object {
          "padding": 8,
        },
        Object {
          "backgroundColor": "plum",
        },
        Object {
          "flex": 1,
        },
      ]
    }
  >
    <Text
      style={
        Object {
          "color": "red",
        }
      }
    >
      Hola
    </Text>
    <View>
      <Text>
        Lorem ipsum
      </Text>
      <Text>
        En un lugar de la Mancha
      </Text>
      <Text>
        de cuyo nombre no quiero acordarme
      </Text>
    </View>
    <View
      style={
        Array [
          Object {
            "backgroundColor": "yellow",
          },
          Object {
            "padding": 8,
          },
          Object {
            "marginTop": "auto",
          },
        ]
      }
    >
      <Text
        style={
          Array [
            Array [
              Object {
                "padding": 10,
              },
              Object {
                "backgroundColor": "gold",
              },
            ],
            Object {
              "color": "red",
              "fontSize": undefined,
            },
          ]
        }
      >
        Esto es una descripción
      </Text>
      <View
        style={
          Array [
            Object {
              "margin": 10,
            },
            Object {
              "backgroundColor": "#d23456",
            },
          ]
        }
      >
        <Text
          style={
            Array [
              undefined,
              Object {
                "color": undefined,
                "fontSize": undefined,
              },
            ]
          }
        >
          Hola muy buenas
        </Text>
        <Text
          style={
            Array [
              Object {
                "fontSize": 45,
                "fontWeight": "bold",
              },
              Object {
                "color": "black",
                "fontSize": undefined,
              },
            ]
          }
        >
          Cómo están ustedes!
        </Text>
        <Image
          source={
            Object {
              "height": 150,
              "uri": "https://placekitten.com/g/300/300",
              "width": 150,
            }
          }
        />
      </View>
    </View>
  </View>
</RCTSafeAreaView>
`;
